src/inhibitors/cooldown.ts

```diff
@@ -14,30 +14,32 @@ botCache.inhibitors.set("cooldown", async function (message, command) {
   const key = `${message.author.id}-${command.name}`;
   const cooldown = membersInCooldown.get(key);
   if (cooldown) {
-    if (cooldown.used + 1 >= (command.cooldown.allowedUses || 1)) {
+    if (cooldown.used >= (command.cooldown.allowedUses || 1)) {
```

Comment:
Why was the +1 here removed?

Reply:
It was removed because the used counter would get increased before checking if it is out of the given max uses range. That is why the user would only only be able to run the command one time before the cooldown gets activated.